src/pages/index.jsx

```diff
@@ -1,10 +1,20 @@
-import React from 'react';
-import { Navbar } from '../components';
-
-const Homepage = () => (
-  <div>
-    <Navbar />
-  </div>
-);
+import React, { useState } from 'react';
+import { DesktopNav, MobileNav, MenuContext, Heading1 } from '../components';
+
+const Homepage = () => {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  const toggleMenuOpen = () => {
+    // eslint-disable-next-line no-unused-expressions
```

Comment:
You can remove the error by directly returning the expression. The single line arrow function declaration thing.

Reply:
Cool

Reply:
Done